zinit.zsh

```diff
@@ -65,6 +65,8 @@ fi
 if [[ -z ${ZINIT[LIST_COMMAND]} ]]; then
     if (( ${+commands[exa]} )); then
         ZINIT[LIST_COMMAND]='exa --color=always --tree --icons -L3'
+    elif (( ${+commands[eza]} )); then
+        ZINIT[LIST_COMMAND]='eza --color=always --tree --icons -L3'
```

Comment:
Maybe check for `eza` before `exa`? If both are installed on the system, we'd probably want to prefer the non-deprecated one.

```suggestion
    if (( ${+commands[eza]} )); then
        ZINIT[LIST_COMMAND]='eza --color=always --tree --icons -L3'
    elif (( ${+commands[exa]} )); then
        ZINIT[LIST_COMMAND]='exa --color=always --tree --icons -L3'
```